### PR TITLE
[UI Fix] Update checkbox styles and make it obvious for issue #2009

### PR DESCRIPTION
**Pull request by Chopinsky**
This PR is trying to fix a styling issue with the "color blind mode" checkbox, as stated in issue #2009. We're going to define a few container CSS classes that will help improve the user interaction with the checkbox, and moving the checkbox up, such that it will be more obvious to users. 

- Before:
image

- After:
image

The updates only concern the web version of the checkbox, the style of the mobile version is left untouched, yet we've moved the checkbox up, and now it's right below the country selection list as well, such that the layout is kept consistent with the web version.

### CODE REVIEW

**Comment by systemcatch** [comment]
Hello @Chopinsky, can you post a full screenshot of the new map for context.

**Comment by Chopinsky** [comment]
Sure, here's the screen shot of the map with the updated checkbox:

image

**Comment by systemcatch** [comment]
@Chopinsky looks good! Let's see what @ovbm thinks.

**Comment by ovbm** [comment]
hey, i like it, could we have a border-radius of 2-4px so that it’s more coherent with the rest of the UI?

**Comment by Chopinsky** [comment]
> hey, i like it, could we have a border-radius of 2-4px so that it’s more coherent with the rest of the UI?

I've added a border radius to `6px` (which is consistent with map's right sidebar toggle buttons), and you can see the effect in the screen shot below. 

image

I've tried `2px` and `4px` options, the checkbox's angle curvature are less obvious, but it might be more consistent with the social media buttons ate the bottom of the left side panel.

File: web/src/styles.css [hunk 1]
```diff
@@ -2184,3 +2280,76 @@ div::-webkit-scrollbar-thumb {
     border-radius: 3px;
 }
 
+/* 
+    custom "color blind mode" checkbox and others
+    adopted with tweeks from source: https://www.w3schools.com/howto/howto_css_custom_checkbox.asp 
+*/
+
+/* The container */
+.container {
```
Comment (ovbm) [inline web/src/styles.css]: I think we should call this checkbox-container. "container" is too general

File: web/views/pages/index.ejs [hunk 2]
```diff
@@ -158,22 +158,23 @@ co2Sub = function(str) {
                         </div>
 
                         <div class="info-text small-screen-hidden">
+                            <p>                                
+                                <label class="container"><%- __('legends.colorblindmode') %>
```
Comment (ovbm) [inline web/views/pages/index.ejs]: class="checkbox-container"

**Comment by ovbm** [comment]
6px looks great! thanks

**Pull request merged**